src/tm/class-wpml-tm-page-builders-field-wrapper.php

```diff
@@ -93,14 +93,35 @@ public function get_string_type() {
 		return $result;
 	}
 
+	/**
+	 * @param string $string_name
+	 *
+	 * @return string
+	 */
+	public static function get_wrap( $string_name ) {
+		$heading = '';
+		if ( strpos( $string_name, '-heading-' ) ) {
+			$name_arr = explode( '-', $string_name );
+			if ( isset( $name_arr[3] ) ) {
+				$heading = $name_arr[3];
```

Comment:
@kagg-design are you sure you want the fourth element of the array?

This method is not covered by tests and I can't see what a possible value of `$string_name´ could be.

Can you please add tests coverage, so you I'll also get an answer to my question?

Reply:
@andreasciamanna test added. We need the 4th element here, as I see no other way to send heading type from Page Builder (like Elementor, for instance), to further processing. Or, as Pierre offers, we need to add one more field to icl_strings table.

Reply:
@Igor-WPML 

I'm still a bit confused.

How exactly something like `title-heading-8e0908e-h2` is created? 

Is it the page builder (e.g. Elementor) creating this string, or is this library (wpml-page-builders)?

Reply:
@andreasciamanna it is created by wpml-page-builders-elementor: https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/63

Reply:
@Igor-WPML you should probably address Bruce's comment on OnTheGoSystems/wpml-page-builders-elementor#63